lib/std/fmt.zig

```diff
@@ -2575,6 +2590,13 @@ pub fn hexToBytes(out: []u8, input: []const u8) ![]u8 {
     return out[0 .. in_i / 2];
 }
 
+test "bytesToHex" {
```

Comment:
Check this out:

```suggestion
test bytesToHex {
```

This communicates that this is the main test case associated with this function :)

There are a lot of other places in std that need to be updated in this manner, so we can do that separately.